depenguinme.sh

```diff
@@ -170,6 +165,25 @@ if ! ip route get 1 >/dev/null 2>&1; then
 	DEPS+=("socat:socat")
```

Comment:
Your current code makes this break (either it won't work due to array undefined, or the array will be overwritten below)

Reply:
awesome thanks @grembo, I missed that